### PR TITLE
Feature: Ringtones

### PR DESCRIPTION
**Pull request by Chuongv**
Ringtones added for both incoming and outgoing calls

### CODE REVIEW

**Comment by dvor** [comment]
Where did you get ringtone? :-) Also for outgoing calls there should be not ringtone but ringback tone.

File: Antidote/RingTonePlayer.m [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- (instancetype)init
+{
+    self = [super self];
+
+    if (! self) {
+        return nil;
+    }
+
+    return self;
```
Comment (dvor) [inline Antidote/RingTonePlayer.m]: No need in empty `init` method.

File: Antidote/RingTonePlayer.m [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+- (void)createAudioPlayer
+{
+    NSString *path = [NSString stringWithFormat:@"%@/%@", [[NSBundle mainBundle] resourcePath], kSoundFilePath];
```
Comment (dvor) [inline Antidote/RingTonePlayer.m]: `NSString *path = [[[NSBundle mainBundle] resourcePath] stringByAppendingPathComponent:kSoundFilePath];`

**Comment by Chuongv** [comment]
>
Where did you get ringtone? :-) Also for outgoing calls there should be not ringtone but ringback tone.

I found the ringtone here http://www.freesound.org/people/Tempouser/sounds/124076/

**Comment by dvor** [comment]
Will you find and add ringback sound as part of this pull request?

**Comment by Chuongv** [comment]
Yep sure thing :)

**Comment by dvor** [comment]
:+1:

**Comment by Chuongv** [comment]
Squashed the commits and have the links to both the ringback and ringtone sounds in the commit message. They are both creative common licenses, so I think we are good there? I'm not a lawyer :cry:

**Comment by dvor** [comment]
I don't know too. :-) Anyway I'm going to check all resources before going to production. I think we are fine by now.

**Pull request merged**